### ISSUE TITLE
Reading some files with NVDA can crash both NVDA and Vs Code

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#118187]
/CC @isidorn 

Issue Type: <b>Bug</b>

I'm unsure if this is an NVDA or a Vs Code bug, so have also cross-posted in nvaccess/nvda#12125

1. Copy the contents of [crash.txt](https://github.com/microsoft/vscode/files/6089611/crash.txt) into a new document in Vs Code
2. With NVDA running, press ctrl+end to go to the bottom, and press up arrow to read upwards.

Expected: it says `}`, as that is the end of the JSON output.

Actual: it read some data that belongs to lines further up. I think this is caused by that underlying chromium bug that prevents long files from being read properly.

More critically, it causes Vs Code to hang and NVDA to also hang as well. The only way to recover is to force kill Vs Code, which could be difficult if NVDA hangs as well.

NVDA version: alpha-21802,5678c8be
VS Code version: Code - Insiders 1.55.0-insider (567b82a0f9c31bd463af64997997730c5d0173cc, 2021-03-05T05:22:52.277Z)
OS version: Windows_NT x64 10.0.19041
Remote OS version: Linux x64 5.4.72-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 PRO 4750U with Radeon Graphics (16 x 1697)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.23GB (18.49GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+Ubuntu/home/dickson/source/api-notarise-formsg --remote=wsl+Ubuntu --crash-reporter-id 1304d64e-abdb-456c-beb6-72c33120f731|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.4.72-microsoft-standard-WSL2|
|CPUs|AMD Ryzen 7 PRO 4750U with Radeon Graphics (16 x 1696)|
|Memory (System)|24.45GB (20.49GB free)|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-containers|ms-|0.163.0
remote-ssh|ms-|0.65.0
remote-ssh-edit|ms-|0.65.0
remote-wsl|ms-|0.54.0
vscode-remote-extensionpack|ms-|0.20.0
vscode-eslint|dba|2.1.14
gitlens|eam|11.3.0
prettier-vscode|esb|5.9.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt602:30263608
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30244315
pythonvsdeb440:30224570
pythonvsded773:30223139
pythonvspyt875:30259475
pythonvsnew554:30265444
pythontb:30258533
vscoresta800cf:30265948
vspre833cf:30267465

```

</details>

**Comment by isidorn** [comment]
@neurrone Thanks a lot for reporting this. I can reproduce. Here are some findings:
* This is unrelated to our change to make `accessibility.pageSize = 2000` by default, even if I change `pageSize` to be `100` I can still reproduce. I believe this was always happening and is not a regression.
* I can reproduce this with just Chrome textArea (that has wrapping disabled) and NVDA. Thus it is an upstream issue and I will comment on the NVDA issue with more details
* I can not reproduce with VoiceOver on macOS

**Comment by Neurrone** [comment]
Thanks for getting on top of this so quickly 🙏. It doesn't trip me up often but when I do have to work on files like this, it is very impactful.

**Comment by Neurrone** [comment]
There's actually another issue that I briefly eluded to in the description; which is that pressing up arrow from the bottom doesn't actually read `}` but instead reads part of lines above it that it shouldn't.

I'd file a separate bug for this, but the crash would make any attempt at investigating and fixing it difficult.

**Comment by jvesouza** [comment]
Basically the same problem happens when using orca. The difference is that the orca does not crash.

**Comment by isidorn** [comment]
@jvesouza you mean the wrong thing gets read? That the `}` does not get actually read?

**Comment by jvesouza** [comment]
@isidorn When I try to read the offending file in VScode, VScode crash but orca does not crash. I need to restart VScode.

**Comment by isidorn** [comment]
@jvesouza thanks. This is due to chrome crashing.

Can everybody please comment on this issue https://bugs.chromium.org/p/chromium/issues/detail?id=1185128

**Comment by isidorn** [comment]
I have verified this is fixed with Chrome 94.0.4606.81.
Thus adding fixed-in-electron15 label.
I have also asked Chrome engineers if they can provide a pointer to a commit that fixed this so we potentially backport the fix.

**Comment by isidorn** [comment]
This should be fixed now in Insiders since we are using Electron 16.
Closing this optimistically and we can reopen if we decide to go back to Electron 14 for this milestone.

@neurrone let us know once you try it out.

**Comment by Neurrone** [comment]
I can verify that it no longer crashes.

However, it seems to get stuck on the line with lots of text once you start arrowing into that area.

But the fact that it no longer crashes is a massive improvement.

**Comment by isidorn** [comment]
@Neurrone great, glad to hear that!
Can you also reproduce getting stock on the line with a lot of text on this set https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

Just try putting that lot of text into the text area. If you can reproduce than it is a Chrome bug and would be great if you can file one here https://bugs.chromium.org/p/chromium/issues/list

**Comment by roblourens** [comment]
We are going back to Electron 13 for February, so I will move this back to March
